web/packages/teleport/src/Roles/Roles.tsx

```diff
@@ -48,11 +49,22 @@ import { RoleList } from './RoleList';
 import templates from './templates';
 import { State, useRoles } from './useRoles';
 
-// RoleDiffProps are an optional set of props to render the role diff visualizer.
+/** Optional set of props to render the role diff visualizer. */
 type RoleDiffProps = {
   roleDiffElement: React.ReactNode;
   updateRoleDiff: (role: Role) => void;
-  errorMessage: string;
+
+  /** @deprecated Use {@link RoleDiffProps.roleDiffAttempt} instead. */
```

Comment:
Nit, but does using `@link` enriches the output in your editor? Because mine just renders it as `{@link RoleDiffProps.roleDiffAttempt}` and we don't generate docs from JSDoc anyway, so if it doesn't do much for other editors either, I'm thinking we could not use `@link`. 😄

I know we do the square brackets thing in Go (e.g. `Use [roleDiffAttempt]`), but arguably it's less disruptive to the surrounding text than `@link`.

Reply:
I'm using VS Code, and it supports @link, but doesn't support square brackets. What do you use? If your one supports square brackets, and this is the convention that we use, perhaps it's about plugin configuration.